apps/angular/src/app/anime-table/anime-table.component.html

```diff
@@ -0,0 +1,43 @@
+@if (animeAll$ | async; as animeAll){
+  <table mat-table [dataSource]="animeAll" class="mat-elevation-z8 anime-table">
+
+    <!-- Image Column -->
+    <ng-container matColumnDef="image">
+      <th mat-header-cell *matHeaderCellDef class="anime-table__title typography_title"> Image </th>
+      <td mat-cell *matCellDef="let anime" class="anime-table__content typography_content"> <img class="anime-table__image" src={{anime.image}} alt={anime.titleJpn}}> </td>
```

Comment:
```suggestion
      <td mat-cell *matCellDef="let anime" class="anime-table__content typography_content"> <img class="anime-table__image" [src]="anime.image" [alt]="anime.titleJpn"> </td>
```
It is better to use property bindings than property interpolation